### DISCUSSION TITLE
Can do at leatst once and exactly once?

### DISCUSSION THREAD

**Discussion by wqzcoder** [risingwavelabs/risingwave#2231] [General]
As we all kone,Flink can do  at leatst once and exactly once,however，Yours is easier to use than it is, with a lower threshold。So  can apply at leatst once and exactly once?

**Comment by yingjunwu** [comment]
Thanks for your interest. RisingWave is designed to support exactly-once delivery. We are considering whether to support weaker delivery guarantees. If you prefer a weaker delivery guarantee such as at-least-once or at-most-once, please let us know! Thanks!
